components/emoji_picker/emoji_picker.tsx

```diff
@@ -1,18 +1,18 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {useRef, useState, useEffect, useCallback, memo, useMemo} from 'react';
-import {FormattedMessage} from 'react-intl';
-import type {FixedSizeList} from 'react-window';
+import React, { useRef, useState, useEffect, useCallback, memo, useMemo } from 'react';
```

Comment:
Could you revert the changes your autoformatter made to the spacing inside the curly brackets in this file? That's causing the linter to fail, and it might also be the cause of some of the conflicts in this file